src/events/cd-user-ticket-list-item.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="cd-user-ticket-list-item">
+    <div class="cd-user-ticket-list-item__details">
+      <header class="cd-user-ticket-list-item__header">
+        <h3 class="cd-user-ticket-list-item__name">
+          {{ event.name }}
+        </h3>
+      </header>
+      <div class="cd-user-ticket-list-item__datetime">
+        <div v-if="!isRecurring" class="cd-user-ticket-list-item__date-timestamp">
+          {{ event.dates[0].startTime | cdDateFormatter }}
+        </div>
+        <div v-else>
+          <div class="cd-user-ticket-list-item__date-series">{{ $t('Next in series:') }}</div>
+          <span class="cd-user-ticket-list-item__date-timestamp">{{ nextStartTime |  cdDateFormatter }}</span>
+        </div>
+        <div class="cd-user-ticket-list-item__times-timestamp">
+          {{ event.dates[0].startTime | cdTimeFormatter }} - {{ event.dates[0].endTime | cdTimeFormatter }}
+        </div>
+      </div>
+    </div>
+    <div v-if="!isPastEvent" class="cd-user-ticket-list-item__view-wrapper">
+      <div v-if="event.eventbriteId">
+        <a :href="event.eventbriteUrl | cdUrlFormatter" target="_blank" class="btn btn-lg btn-primary cd-user-ticket-list-item__view">{{ $t('See Details and Book') }}</a>
+      </div>
+      <div v-if="hasApplications" class="cd-user-ticket-list-item__view-applications">
+        <h4 class="cd-user-ticket-list-item__view-application-title"><i class="fa fa-ticket"></i>{{ $t('Tickets') }}</h4>
+        <div v-for="application in applications">
+          <cd-attendee :application="application" :session="sessions[application.sessionId]" :ticket="tickets[application.ticketId]" :user="users[application.userId]"></cd-attendee>
+        </div>
+        <button tag="button"
+          @click="cancel()"
+          class="btn btn-lg cd-user-ticket-list-item__view-cancel-button">
+          {{ $t('Cancel ticket', applications.length) }}</button>
+      </div>
+      <!-- NOTE : What about awaiting approval tickets ?-->
```

Comment:
Yeah, what about it? :P

Reply:
Well, we should prob display them differently. But that's another story? :)